src/main/kotlin/no/nav/klage/oppgave/api/mapper/BehandlingMapper.kt

```diff
@@ -200,29 +161,32 @@ class BehandlingMapper(
         }
     }
 
-    private fun SakenGjelder.mapToView(): BehandlingDetaljerView.SakenGjelderView {
-        if (erPerson()) {
-            val person = pdlFacade.getPersonInfo(partId.value)
+    private fun getSakenGjelder(sakenGjelder: SakenGjelder): BehandlingDetaljerView.SakenGjelderView {
```

Comment:
Hvorfor ble signaturen endret? Den burde i tilfelle kanskje kalles for noe som `getSakenGjelderView(...)`.

Reply:
Enig, det skal være med `view` der. Grunnen til at jeg endret er at metoden henter info fra eksterne punkter (pdl eller ereg), og da tenker jeg at det blir feil å kalle det mapping. Vet ikke om du er med på den tanken?

Reply:
Muligens litt med på tanken på navnebyttet. Men ikke sikker :) Men å endre fra extension function til "classic" getter vet jeg ikke.